docs/pages/identity-governance/access-lists/reviewing-access-requests.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Reviewing Access Requests
+description: Learn how Access List owners review Access Requests
+tags:
+ - get-started
+ - identity-governance
+ - privileged-access
+ - access-lists
+ - access-requests
+---
+
+Access list owners can be automatically assigned as suggested reviewers to
+resource-based access requests that include resources granted by their access
+list. In this guide we will walk through an example and the configuration of
+this use-case.
+
+## Prerequisites
+
+- Teleport cluster with a connected resource e.g. an SSH node.
+- Teleport user (`alice` in this guide) with an `editor` or equivalent administrative role.
+- Teleport user (`bob` in this guide) acting as a low-privileged requester user.
+
+## Step 1/5. Create roles
+
+First, let's create 3 roles:
+
+- Role that grants access to SSH nodes with a particular label
```

Comment:
Nit: mention the label `env: prod` here ( instead of below in line 31) so reference to the "particular label" is more apparent.